src/App.vue

```diff
@@ -1,18 +1,18 @@
 <template>
+  <BaseHeader />
   <router-view></router-view>
 </template>
 
 <script lang="ts">
 import { defineComponent } from 'vue';
+import BaseHeader from '@/components/base/BaseHeader.vue';
 
```

Comment:
i think we already got this component in router

Reply:
As I said on daily meeting 20.09, we should move header to app component and delete it from other places.